tony-history-server/app/utils/HdfsUtils.java

```diff
@@ -24,6 +25,12 @@
 public class HdfsUtils {
   private static final Logger.ALogger LOG = Logger.of(HdfsUtils.class);
 
+  /**
+   * Check to see if a HDFS path is valid.
```

Comment:
Please define what "valid" means. It looks like this method just tests for path existence, in which case, why do we need this method? Why not just call `fs.exists(filePath)` directly?

Reply:
You're correct, but imo for the sake of code modularity, we should have the method. Calling `fs.exists` directly would require the try/catch block every time you call it, which makes the caller method more verbose than it should be.

Reply:
Ah, I see. Avoiding try/catch blocks is a reasonable reason for adding this method.

Can we still update the description and replace `is valid` with `exists`? I think that's clearer. Also, I suggest renaming the method `pathExists`.